### PR TITLE
[Quick Tour] Fixed things found by the docbot

### PR DESCRIPTION
**Pull request by wouterj**
Found some time to get the docbot working again. Before showing it in action, we have to make sure the current docs comply with its standards. That means fixing a lot of too long lines, we have a very strict bot now :)

Besides that, this PR also fixes some serial comma, wrong title case and other wrong usage.

### CODE REVIEW

File: quick_tour/the_controller.rst [hunk 1]
```diff
@@ -309,9 +314,9 @@ from any controller::
         $foo = $session->get('foo', 'default_value');
     }
 
-You can also store "flash messages" that will auto-delete after the next request.
-They are useful when you need to set a success message before redirecting the
-user to another page (which will then show the message)::
+You can also store "flash messages" that will auto-delete after the next
+request.  They are useful when you need to set a success message before
```
Comment (javiereguiluz) [inline quick_tour/the_controller.rst]: Minor error: two white spaces after `request.`

File: quick_tour/the_controller.rst [hunk 2]
```diff
@@ -332,8 +337,8 @@ And you can display the flash message in the template like this:
...
 
-That's all there is to it, and I'm not even sure you'll have spent the full
-10 minutes. You were briefly introduced to bundles in the first part, and all the
-features you've learned about so far are part of the core framework bundle.
-But thanks to bundles, everything in Symfony can be extended or replaced.
-That's the topic of the :doc:`next part of this tutorial <the_architecture>`.
+That's all there is to it and I'm not even sure you'll have spent the full
+10 minutes. You were briefly introduced to bundles in the first part and
+all the features you've learned about so far are part of the core framework
+bundle.  But thanks to bundles, everything in Symfony can be extended or
```
Comment (javiereguiluz) [inline quick_tour/the_controller.rst]: Minor error: two white spaces after `bundle.`

File: quick_tour/the_view.rst [hunk 3]
```diff
@@ -50,34 +50,41 @@ Below is a minimal template that illustrates a few basics, using two variables
         </body>
     </html>
 
-To render a template in Symfony, use the ``render`` method from within a controller.
-If the template needs variables to generate its contents, pass them as an array
-using the second optional argument::
+To render a template in Symfony, use the ``render`` method from within a
+controller.  If the template needs variables to generate its contents, pass
```
Comment (javiereguiluz) [inline quick_tour/the_view.rst]: Minor error: two white spaces after `controller.`

File: quick_tour/the_view.rst [hunk 4]
```diff
@@ -269,12 +278,12 @@ extensible way.
...
-the basics is easy, and you will soon learn that this simplicity is hidden
+the basics is easy and you will soon learn that this simplicity is hidden
 under a very flexible architecture.
 
-But I'm getting ahead of myself. First, you need to learn more about the controller
-and that's exactly the topic of the :doc:`next part of this tutorial <the_controller>`.
-Ready for another 10 minutes with Symfony?
+But I'm getting ahead of myself. First, you need to learn more about the
+controller and that's exactly the topic of the :doc:`next part of this tutorial
+<the_controller>`.  Ready for another 10 minutes with Symfony?
```
Comment (javiereguiluz) [inline quick_tour/the_view.rst]: Minor error: two white spaces after `<the_controller>.`

**Comment by javiereguiluz** [comment]
@WouterJ excellent work! Congrats! I'm really looking forward to see the bot in action.

**Comment by WouterJ** [comment]
Thanks for your quick review, @javiereguiluz! I've added this to the docbot reviewers and found a couple more occurences. Fixed them & pushed again :heavy_check_mark:

**Comment by stof** [comment]
@WouterJ shouldn't you fix the 2.3 branch first, and then fix 2.7 after merging 2.3 into it ?

**Comment by WouterJ** [comment]
@stof thanks for noticing it. I've done the changes against 2.3, but it seems that I choose the wrong branch for the PR.

**Comment by WouterJ** [comment]
@javiereguiluz you can try it yourself with a (very on-the-edge) release of the docbot: https://github.com/WouterJ/docbot

**Comment by WouterJ** [comment]
I've merged this one now. If anyone spots something wrong, please create a PR fixing it :)

**Pull request closed without merging**